dandi/tests/test_girder.py

```diff
@@ -0,0 +1,37 @@
+import pytest
+from ..exceptions import LockingError
+from .. import girder
+
+
+def test_lock_dandiset(local_docker_compose_env):
+    DANDISET_ID = "000001"
+    client = girder.get_client(local_docker_compose_env["instance"].girder)
+    resp = client.get(f"dandi/{DANDISET_ID}/lock/owner")
+    assert resp is None
+    with client.lock_dandiset(DANDISET_ID):
+        resp = client.get(f"dandi/{DANDISET_ID}/lock/owner")
+        assert resp is not None
+    resp = client.get(f"dandi/{DANDISET_ID}/lock/owner")
+    assert resp is None
+
+
+def test_lock_dandiset_error_on_nest(local_docker_compose_env):
+    DANDISET_ID = "000001"
+    client = girder.get_client(local_docker_compose_env["instance"].girder)
+    with client.lock_dandiset(DANDISET_ID):
+        with pytest.raises(LockingError) as excinfo:
+            with client.lock_dandiset(DANDISET_ID):
+                pass
```

Comment:
since not intended to be covered, 
```suggestion
                pass  # pragma: no cover
```

Reply:
Done.